window_manager/src/MsWindow.hpp

```diff
@@ -0,0 +1,135 @@
+#ifndef RDMN_MS_WINDOW_HPP
+#define RDMN_MS_WINDOW_HPP
+
+#define WIN32_LEAN_AND_MEAN
```

Comment:
Already present in `.bazelrc`, so not needed.

Reply:
Removed